principles.md

```diff
@@ -0,0 +1,34 @@
+# Status Principles
+
+The goal of Status is widespread adoption of the decentralized web. Our challenge is achieving mass adoption while staying true to our principles outlined below.
+
+**Liberty**
+We believe in the sovereignty of individuals. As a platform that stands for the cause of personal liberty, we aim to maximize social, political, and economic freedoms.
+
+**Censorship resistance**
+We enable free flow of information. No content is under surveillance. We abide
+by the cryptoeconomic design principle of censorship resistance.
+
+**Security**
+We don't compromise on security when building features. We use state-of-the-art technologies, and research new security methods and technologies to make strong security guarantees.
+
+**Privacy**
+Privacy is the power to selectively reveal oneself to the world. For us, it's essential to protect privacy in both communications and transactions, as well as to preserve the right to total anonymity.
+
+**Transparency**
+We strive for complete openness and symmetry of information, and have no border between our core contributors and our community. We are frank about our shortcomings, especially when making short-term tradeoffs in service of our long-term goals.
+
+**Free culture**
```

Comment:
My only tiny complaint: `Free culture` is evocative of something much different to me than this. I would find it easier to understand `Free use.`

Reply:
Isn't `Libre` commonly used for this?

Reply:
Free culture is referring specifically to https://en.wikipedia.org/wiki/Free-culture_movement, but changing it to Free use for now based on demand. If someone strongly disagrees with this, we can have a debate.